docs/spfx/use-aadhttpclient.md

```diff
@@ -206,7 +206,7 @@ Global and SharePoint administrators can also use the SharePoint Online Manageme
   - **ResourceId**: The objectId of the resource service principal to which access has been granted.
   - **Scope**: The value of the scope claim that the resource application should expect in the OAuth 2.0 access token.
 
-- To *revoke a previously granted permission*, use the **Revoke-SPOTenantServicePrincipalPermission -ObjectId <String>** cmdlet. In the **ObjectId** parameter, you should specify the objectId of the grant that you want to revoke, which you can obtain by using the **Get-SPOTenantServicePrincipalPermissionGrants** cmdlet.
+- To *revoke a previously granted permission*, use the **Revoke-SPOTenantServicePrincipalPermission -ObjectId &lt;String&gt;** cmdlet. In the **ObjectId** parameter, you should specify the objectId of the grant that you want to revoke, which you can obtain by using the **Get-SPOTenantServicePrincipalPermissionGrants** cmdlet.

```

Comment:
Correct markdown escaping of angled brackets should be prefixed with`\`; don't change to the HTML encoding.